racun-service/src/main/java/rs/edu/raf/banka/racun/service/impl/SredstvaKapitalService.java

```diff
@@ -158,7 +160,7 @@ public List<KapitalPoTipuHartijeDto> getStanjeJednogTipaHartije(String token, St
                 kapitalPoTipuHartijeDto.setCena(akcijePodaciDto.getPrice());
                 kapitalPoTipuHartijeDto.setKodValute(berzaDto.getKodValute());
 
-                ForexPodaciDto forexPodaciDto = this.getForex(token, berzaDto.getKodValute());
+                ForexPodaciDto forexPodaciDto = this.getForex(token, berzaDto.getKodValute(), "RSD");
```

Comment:
Zašto je ovde RSD, a ne USD?

Reply:
To je onaj drugi prikaz (sve hartije za jedan tip) i tu prikazujemo cenu u dinarima i u originalnoj valuti.